### PR TITLE
Add orbit design tokens

### PR DESCRIPTION
**Pull request by tbergq**

### CODE REVIEW

File: app/MMB/src/components/alert/__tests__/__snapshots__/Alert.test.js.snap [hunk 1]
```diff
@@ -43,12 +43,12 @@ exports[`Alert renders danger type 1`] = `
 <RenderAlert
   containerStyle={
     Object {
-      "backgroundColor": "rgba(208, 2, 27, 0.15)",
```
Comment (RobinCsl) [inline app/MMB/src/components/alert/__tests__/__snapshots__/Alert.test.js.snap]: I know it's a snap, but we don't have any transparency anymore, right?

Comment (tbergq) [reply app/MMB/src/components/alert/__tests__/__snapshots__/Alert.test.js.snap]: Well, when I had the meeting with the orbit designer, he told me we had the wrong colors on our alerts

File: app/hotels/src/navigation/allHotels/SearchResultsHeader.js [hunk 2]
```diff
@@ -50,7 +50,7 @@ export default function SearchResultsHeader(props: Props) {
                   renderOnNarrow={
                     <MapHeaderButton
                       onPress={props.goToMap}
-                      iconColor={Color.ink.light}
+                      iconColor={defaultTokens.paletteInkDark}
```
Comment (RobinCsl) [inline app/hotels/src/navigation/allHotels/SearchResultsHeader.js]: It was light before? Is it a typo?

Comment (tbergq) [reply app/hotels/src/navigation/allHotels/SearchResultsHeader.js]: Yes! I will fix it right away

File: app/hotels/src/singleHotel/roomList/RoomRowTitle.js [hunk 3]
```diff
@@ -73,3 +59,18 @@ export default (createFragmentContainer(
...
 ): React.ComponentType<ContainerProps>);
+
+const styles = StyleSheet.create({
+  title: {
+    fontSize: 14,
+    color: defaultTokens.colorTextAttention,
+  },
+  icon: {
+    fontSize: 10,
+    color: defaultTokens.colorTextSecondary,
```
Comment (RobinCsl) [inline app/hotels/src/singleHotel/roomList/RoomRowTitle.js]: `colorIconSecondary`?

File: packages/navigation/index.js [hunk 4]
```diff
@@ -11,6 +11,8 @@ import {
   type StylePropType,
   type DimensionType,
 } from '@kiwicom/mobile-shared';
+// Seems like we get circular dependencies if I import defaultTokens from mobile-shared, don't know why though
```
Comment (RobinCsl) [inline packages/navigation/index.js]: Maybe create an issue to investigate this further at some later point?

File: packages/shared/src/icons/TextIcon.js [hunk 5]
```diff
@@ -47,3 +40,11 @@ class TextIcon extends React.Component<Props> {
 }
 
 export default TextIcon;
+
+const styleSheet = StyleSheet.create({
+  icon: {
+    fontFamily: 'spfont', // font with all the icons (see Fontastic)
```
[the remark this answers is not in the document]
Comment (tbergq) [reply packages/shared/src/icons/TextIcon.js]: That is the plan yes

File: app/MMB/src/components/alert/__tests__/__snapshots__/Alert.test.js.snap (hunk 1 above)
Comment (RobinCsl) [reply app/MMB/src/components/alert/__tests__/__snapshots__/Alert.test.js.snap]: Okay, let's do as the designer says.

**Pull request merged**